### DISCUSSION TITLE
PyTorch-Ignite sticker for our community

### DISCUSSION THREAD

**Discussion by vfdev-5** [pytorch/ignite#1517] [General]
We would  ❤️  to acknowledge your awesome community and contributors with our logo die cut stickers : 

![ignite-sticker](https://user-images.githubusercontent.com/2459423/106633001-a4cb3e80-657e-11eb-8419-68c5e65288f9.jpg)

Sticker size: 75mm x 37.5mm

If you would like to have your 3 stickers, please send your post address to contact@pytorch-ignite.ai (if you do not have any reply on your email, please ping me here).

@gruebel and @trsvchn , you are more then welcome for that 🥇, please let us know where we could send them to you :)

---

EDIT:
For your information, we'll remove all collected addresses once the stickers are sent.

**Comment by tarasio-mirror** [comment]
Hi, @vfdev-5 !
Just received my PyTorch-Ignite stickers. Going to set my laptop on fire!)
Many thanks! :)

![photo_2021-02-24_20-21-20](https://user-images.githubusercontent.com/30748114/109047964-270ed480-76df-11eb-8de2-fa193eb97967.jpg)

**Reply by vfdev-5** [reply]
Hi @trsvchn , glad that they could arrive without issues !

**Comment by EricZimmermann** [comment]
Are these still available?

**Reply by sdesrozis** [reply]
I don't know, but this seems like a good collector to have ;)

**Reply by vfdev-5** [reply]
@EricZimmermann yes, they are still available. If you would like to have your 3 stickers, please send your post address to contact@pytorch-ignite.ai or by a DM on discord.
